testdata/imports/compose_enums.txtar

```diff
@@ -77,19 +79,21 @@ export interface Compose {
   localstr: LocalEnum;
   localstrd: LocalEnumD;
   localstrover: LocalEnumD;
+  union: dep.DepEnumNumericD;
```

Comment:
Considering the definition is:

```
Compose: {
    ...
    union: (dep.DepEnumNumericD & 2) | (dep.DepEnumNumericD & 3)
    ...
} @cuetsy(kind="interface")
```


Shouldn't this be like: `union: dep.DepEnumNumericD.Two | DepEnumNumericD.Three` ? 🤔

Reply:
This is not working in this PR. Unions are fixed in the other PR 😅.

Reply:
Great, then! 🚀